crates/jstzd/tests/jstzd_test.rs

```diff
@@ -122,20 +113,23 @@ async fn jstzd_test() {
     })
     .await;
     assert!(node_destroyed);
-
     let baker_destroyed = retry(30, 1000, || async {
         Ok(!run_ps().await.contains("octez-baker"))
     })
     .await;
     assert!(baker_destroyed);
-
+    let rollup_destroyed = retry(30, 1000, || async {
```

Comment:
This might not always work. See #664.

Reply:
i will use the health check method for now and can use the approach later